github/ci/prow-sig-scale-workloads/hack/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source $(dirname "$0")/common.sh
+
+main(){
+    setup
+
+    cd ${BASE_DIR}
+
+    ansible-playbook -i inventory/prow-performance-workloads/hosts.yaml  --become --become-user=root --private-key ~/.ssh/id_ed25519 bootstrap.yml
```

Comment:
Is the ssh an `ed25519` key? If it's an rsa (seems like it looking at the private part) we should name it accordingly.

Reply:
It is an `rsa`. I will update it